build/lib/pgBackRestBuild/Config/Data.pm

```diff
@@ -1826,6 +1827,12 @@ my %hConfigDefine =
         &CFGDEF_COMMAND => CFGOPT_REPO_TYPE,
     },
 
+    &CFGOPT_REPO_AZURE_ENDPOINT =>
+    {
+        &CFGDEF_INHERIT => CFGOPT_REPO_AZURE_HOST,
+        &CFGDEF_DEFAULT => 'blob.core.windows.net',
```

Comment:
We usually have a default constant for string defaults (the times we don't we usually also have an ALLOW_LIST). My concern is in testing: if we use a hardcoded value instead of a constant (that would also be used here) then if we accidentally change this value, the tests will happily still work even though the default is now wrong. So I'd like to recommend we use something like CFGOPTVAL_REPO_ENDPOINT_DEFAULT or CFGOPTVAL_DEFAULT_REPO_ENDPOINT or something.

Reply:
If we did put a constant here we'd only be able to use it in Perl. And it's the default so it would never get used.

I'm still planning to move this file into yaml so the constant would be useless then.

Reply:
So you're saying none of our string defaults are in C? I thought we did some sort of auto translation for these types of things from perl into c...

Reply:
That's what I'm saying. I've been meaning to write code to create these constants in C but it has not happened yet. I'm loathe to add any code to the Perl version of the code generator.